processor/src/operations/io_ops.rs

```diff
@@ -73,6 +95,39 @@ impl Process {
         Ok(())
     }
 
+    /// Stores an element from the stack into the specified memory address.
+    ///
+    /// The operation works as follows:
+    /// - The memory address is popped off the stack.
+    /// - The top stack item is saved into the specified memory address. The item is not
+    ///   removed from the stack.
+    ///
+    /// Thus, the net result of the operation is that the stack is shifted left by one item.
+    pub(super) fn op_mstore(&mut self) -> Result<(), ExecutionError> {
+        // get the address from the stack and build the word to be saved from the stack values
+        let addr = self.stack.get(0);
+
+        // get the word stored in the memory
+        let unmodified_word = self.memory.get_value(addr.as_int());
+
+        // if we did not access the memory return [0, 0, 0, 0] since initially the memory is initialized with zeros
+        let mut word = match unmodified_word {
+            Some(word) => word,
+            None => [Felt::new(0), Felt::new(0), Felt::new(0), Felt::new(0)],
+        };
```

Comment:
You can probably simplify this with:
```Rust
let word = self.memory.get_value(addr.as_int()).unwrap_or([Felt::ZERO; 4]);
```

Reply:
And maybe replace `[Felt::ZERO; 4]` with `INIT_MEM_VALUE` - since we already have it defined.